.github/workflows/ci.yml

```
@@ -57,6 +57,13 @@ jobs:
         with:
           java-version: '17'
           distribution: 'corretto'
+      - uses: trappar/turborepo-remote-cache-gh-action@main
+        with:
+          storage-provider: s3
+          storage-path: ${{ secrets.REMOTE_CACHE_BUCKET }}
```

Use remote cache bucket name directly
```suggestion
          storage-path: smithy-typescript-remote-cache
```